support-frontend/app/controllers/Application.scala

```diff
@@ -300,13 +301,12 @@ class Application(
     ).withSettingsSurrogateKey
   }
 
-  def products() = Action.async { implicit request =>
-    wsClient
-      .url(
-        "https://raw.githubusercontent.com/guardian/support-service-lambdas/0b031ea5821c95a7f7c59e45951d2d1f0bebed9d/modules/product/src/prodCatalogMapping.json",
-      )
-      .get()
-      .map(response => Ok(response.body).withHeaders("Cache-Control" -> "max-age=30"))
+  @deprecated("We will remove this once we get the data to the client not via HTTP")
+  def products() = Action { implicit request =>
+    val isTestUser = testUserService.isTestUser(request)
+    val cachedProductCatalogService = cachedProductCatalogServiceProvider.forUser(isTestUser)
+
+    Ok(cachedProductCatalogService.get().toJson).withHeaders("Cache-Control" -> "no-cache, private")
```

Comment:
we have a `NoCacheAction` which saves having to do this explicitly

Reply:
Fab, added!